pkg/cluster/acrtoken.go

```diff
@@ -56,3 +61,51 @@ func (m *manager) ensureACRToken(ctx context.Context) error {
 
 	return nil
 }
+
+func (m *manager) updateACRToken(ctx context.Context) error {
+	// we do not want to rotate tokens in local development
+	// TODO: check for using rh-dev-env-pull on arointsvc along with or instead of local development
+	if m.env.IsLocalDevelopmentMode() {
+		return nil
```

Comment:
I went ahead and tested this with these lines commented out. I got the following:

`ERRO[2023-04-11T10:28:58-04:00]pkg/backend/openshiftcluster.go:253 backend.(*openShiftClusterBackend).updateAsyncOperation.func1() parsing failed for . Invalid resource Id format  client_principal_name= client_request_id= component=backend correlation_id= request_id=078710e5-8660-46b2-b51c-c8bccacce054 resource_group=v4-eastus resource_id=/subscriptions/dev-subs-id/resourcegroups/v4-eastus/providers/microsoft.redhatopenshift/openshiftclusters/cadenaro resource_name=cadenaro subscription_id=dev-subs-id`

It worked on `master` branch fine - It doesn't look to be complaining about the cluster resource ID - maybe it's ACR resource ID?